README.md

```diff
@@ -34,10 +34,40 @@ The `.env` file is used by flask to set environment variables when running `flas
 Find the value for the following keys:
 Set the  .env file.
 
-###Running the todo-terraform-app in azure
-Step 1: You have to have an azure account
-Step 2: Look for the todo-terraform-app under AmericanExpress21Group1_KamolSaha_ProjectExercise
-Step 3: Select the app and the browser should be available
+###Running the todo-app in kubernetees#######
+Step 1: Run minikube start in an admin terminal to spin up your minikube cluster to run kubernetees in your local
+
+Step 2: Build the image with the following command docker build --target production --tag todo-app:prod .
+
+Step 3: Run minikube image load todo-app:prod 
+
+Step 4: Run kubectl apply -f to-do-app-secrets.yaml     
```

Comment:
As I mentioned on that secrets file, you must not commit secrets to GitHub. You can either:
- Remove that secrets.yml file from git and instead tell the reader how to create that file.
- Or you can also create secrets from the command line, skipping creating a file altogether. `kubectl create secret generic foobar --from-literal=my_secret=secret_value`